src/pages/country/component.js

```diff
@@ -3,19 +3,35 @@ import PropTypes from 'prop-types';
 
 import Header from 'components/header';
 import Hero from 'components/hero';
+import Spinner from 'components/spinner';
 import Navigation from 'components/navigation';
-import TabsInfo from './constants';
+import CardInfo from 'components/card';
+import Summary from 'components/summary';
+import { fetchCategories } from 'components/widget/utils';
+import useAxios from 'axios-hooks';
 
 const CountryPage = ({ iso, current }) => {
-  const infoPage = TabsInfo.find((info) => info.category === current);
+  const [{ data, loading }] = useAxios(fetchCategories());
+  const tabs = data && data.rows ? data.rows : null;
+
+  const currentTab = tabs ? tabs.find((tab) => tab.slug === current) : null;
+  const name = currentTab ? currentTab.name : null;
+  const slug = currentTab ? currentTab.slug : null;
+
   return (
     <div className="l-country">
       <Header />
       <Hero iso={iso} />
-      <div className="country-content">
-        <Navigation tabs={TabsInfo} iso={iso} currentTab={current} />
-        {infoPage.content}
-      </div>
+      {loading ? (
```

Comment:
It is very important to check loading and data, it is not enough to check only loading.